tests/readonly_app/views.py

```diff
@@ -14,4 +14,4 @@ def test_view(request):
         content="OK" if valid else "Fail")
 
 
-urlpatterns = [url("^", test_view)]
+urlpatterns = [path("^", test_view)]
```

Comment:
Since the `path` is used here is `^` is needed here?

Reply:
Done in https://github.com/peopledoc/django-readonly-field/pull/19/commits/19ed742fdb2b067b433aefc9aeeac8a7735d29ae